src/chrome/content/rules/LinuxFoundation.xml

```diff
@@ -44,9 +36,7 @@
 	<target host="collabprojects.linuxfoundation.org" />
 	<target host="developerbugs.linuxfoundation.org" />
 	<target host="events.linuxfoundation.org" />
-	<target host="eventsstg.linuxfoundation.org" />
 	<target host="identity.linuxfoundation.org" />
-	<target host="ldn.linuxfoundation.org" />
```

Comment:
Ok, we can leave them in. we have tens if not hundreds of such redirect-in-entirety subdomains, so it seemed odd to have a rule just for one of them.  I'll get it an updated patch written.